docs/bank-connect/rest-api.md

```diff
@@ -135,7 +135,8 @@ On fetching information successfully, the response would be of the following for
             "account_number": "Account Number Extracted",
             "account_category": "individual",
             "credit_limit": 0,
-            "bank": "axis"
+            "bank": "axis",
+            "last_updated": "2023-03-04T07:37:27.976531Z"
```

Comment:
put an example with 24 hour time everywhere for more clarity

Reply:
is this needed? the time format is yyyy-MM-ddTHH:mm:ss.SSSZ. It is actually a 24H format

Reply:
if you have mentioned format its okay